integration/actions_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build integration
-// +build integration
+//go:build actions
```

Comment:
could you add an envvar to skip the execution of the `context` action?
that why we can automate the execution of that tests only if the CLI version under test support the `context` command